internal/backend/db/dbgorm/envs.go

```diff
@@ -39,9 +39,12 @@ func (db *gormdb) CreateEnv(ctx context.Context, env sdktypes.Env) error {
 	return translateError(db.createEnv(ctx, e))
 }
 
+func (db *gormdb) deleteEnvs(ctx context.Context, ids []string) error {
+	return db.db.WithContext(ctx).Where("env_id IN ?", ids).Delete(&scheme.Env{}).Error
```

Comment:
if you'll change to use scopes, you can reuse the function here I stated above

Reply:
see my note for the scopes